index.js

```diff
@@ -781,7 +781,12 @@ function processPaths(container, containerName, options, requestBodyCache, opena
             if ((common.httpVerbs.indexOf(method) >= 0) || (method === 'x-amazon-apigateway-any-method')) {
                 let op = path[method];
 
-                if (op.parameters && Array.isArray(op.parameters)) {
+                if (op.parameters === null) {
+                    delete op.parameters;
+                    continue;
```

Comment:
Is the `continue` wanted / necessary? Which loop is it skipping out of? Should we not delete the op.parameters outside of this `if` test (for the http method)?